tests/integration/sources/test_rpm.py

```diff
@@ -0,0 +1,110 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2022 Canonical Ltd.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License version 3 as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import platform
+import shutil
+import subprocess
+import textwrap
+from pathlib import Path
+
+import pytest
+import yaml
+
+import craft_parts
+from craft_parts.actions import Action
+from craft_parts.steps import Step
+
+
+@pytest.fixture
+def sample_rpm(tmp_path: Path) -> Path:
+    """
+    Create a basic .rpm file and return its path.
+    """
+    rpm_dir = tmp_path / "sample"
+    rpm_dir.mkdir()
+
+    # Add spec structure
+    spec_dir = rpm_dir / "SPECS"
+    spec_dir.mkdir()
+    spec_file = spec_dir / "sample.spec"
+    spec_file.write_text(
+        textwrap.dedent(
+            """
+            Name: sample
+            Version: 1.0
+            Release: 0
+            Summary: A sample package
+            License: GPL
+            
+            %description
+            A little sample package!
+            
+            %install
+            mkdir -p %{buildroot}/etc
+            bash -c "echo Sample contents > %{buildroot}/etc/sample.txt"
+            
+            %files
+            /etc/sample.txt
+            """
+        )
+    )
+
+    # Define paths wo we don't litter the system with RPM stuff for this.
+    rpmbuild_params = [
+        f"--define=_topdir {rpm_dir}/build",
+        f"--define=_dbpath {rpm_dir}/rpmdb",
+        f"--define=_var {tmp_path}/var",
+        f"--define=_tmppath {tmp_path}/tmp",
+    ]
+
+    subprocess.run(
+        ["rpmbuild", "-bb", "--verbose"] + rpmbuild_params + [str(spec_file)],
+        check=True,
+        text=True,
+        capture_output=True,
+    )
+
+    # The asterisks in this glob allow this to be
```

Comment:
which glob?

Reply:
The one I ended up removing from the test anyway! Fixed :-)